packages/cli/src/commands/create.ts

```diff
@@ -20,6 +22,8 @@ const register: (program: any) => any = (program) => program
   .action(action);
 
 async function action(contractFullName: string, options: any): Promise<void> {
+  const zosversion = await ZosPackageFile.getZosversion();
+  if (!await willMigrateProjectIfNeeded(zosversion)) return;
```

Comment:
I would use sth more friendly like `hasToMigrateProject` or so, not a blocker tho.

Reply:
Very nit, but we can move this behavior into `/bin/options.ts` doing sth like:
```js
program.Command.prototype.withMigrateAction = function(): Command {
  return this.action(() => {
     const zosversion = await ZosPackageFile.getZosversion();
     if (!await willMigrateProjectIfNeeded(zosversion)) process.exit(0);
  });
};
```

Reply:
I'm not confortable with the `process.exit(0)` thing, but it's definitely simpler and clean.